DatadogSDKBridge/Classes/Implementation/DdRumImplementation.swift

```diff
@@ -119,13 +138,47 @@ internal class DdRumImplementation: DdRum {
     }
 
     func stopResource(key: NSString, statusCode: Int64, kind: NSString, timestampMs: Int64, context: NSDictionary) {
-        nativeRUM.stopResourceLoading(resourceKey: key as String, statusCode: Int(statusCode), kind: RUMResourceType(from: kind as String), size: nil, attributes: attributes(from: context, with: timestampMs))
+        let mutableContext = NSMutableDictionary(dictionary: context)
+        if let resourceTimings = mutableContext.object(forKey: Self.resourceTimingsKey) as? [String: Any] {
```

Comment:
looks good to me but maybe you could actually extract this logic in a separated function

Reply:
yes, most probably I will extract it as a private `addResourceTimings` function.